internal/receiver/smartagentreceiver/README.md

```diff
@@ -19,7 +19,11 @@ should be used.
 [Collector processors](https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/README.md).
 1. Monitors with [dimension property and tag update
 functionality](https://dev.splunk.com/observability/docs/datamodel#Creating-or-updating-custom-properties-and-tags)
-require an associated `metadataClients` field that references the name of the SignalFx exporter you are using in your
+require an associated `dimensionClients` field that references the name of the SignalFx exporter you are using in your
+pipeline.
+1. Monitors with [event-sending
+functionality](https://dev.splunk.com/observability/docs/datamodel/ingest#Send-custom-events)
+require an associated `eventClients` field that references the name of the SignalFx exporter you are using in your
```

Comment:
Maybe for a later PR, but would it be a better experience to list out monitors that support properties and events explicitly in this README?